src/helpers/map.js

```diff
@@ -93,25 +84,21 @@ export default {
      * @returns {Object} the active item
      */
     getActiveItem: function getActiveItem(map) {
-        var parts = this.getParts(map),
-            result = {};
-
-        _.forEach(parts, function(part) {
-            var sections = part.sections;
-            if (sections) {
-                _.forEach(sections, function(section) {
-                    if (section.active) {
-                        const items = section.items;
-                        _.forEach(items, function(item) {
-                            if (item.active) {
-                                result = item;
-                            }
-                        });
+        const parts = this.getParts(map) || {};
+
+        for (let part of Object.values(parts)) {
+            const sections = part.sections || {};
+            for (let section of Object.values(sections)) {
```

Comment:
does it make sense to continue if `sections` is an empty object?

Reply:
It won't continue, because empty object has no values. But I will add a condition instead.